README.md

```diff
@@ -1 +1,122 @@
-# OME Files Benchmarking scripts
+# OME Files Benchmark
+
+## Introduction
+
+The current repository includes a set of benchmarks used to test the following
+software:
+
+-   [OME Files 0.3.1](http://downloads.openmicroscopy.org/ome-files-cpp/0.3.1/)
+-   [Bio-Formats 5.3.4](http://downloads.openmicroscopy.org/bio-formats/5.3.4/)
+    both using the Java Virtual Machine and the
+    [Jace C++ bindings](https://github.com/ome/bio-formats-jace)
+
+The benchmark suite has been executed under Windows Server 2008 and Ubuntu
+16.04 with the exception of the JACE C++ bindings which we have only built
+under Ubuntu 16.04 using JDK 7.
+
+## Benchmark datasets
+
+The reference datasets used for the benchmark are compose of three public OME-TIFF filesets representing different aspects of the OME Data model:
```

Comment:
I think compose could be dropped; could be comprise.  But could also simply be "Three public reference OME-TIFF datasets were used, representing..."